src/execution/execute.js

```diff
@@ -760,16 +757,14 @@ export function resolveFieldValueOrError<TSource>(
     // TODO: find a way to memoize, in case this field is within a List type.
     const args = getArgumentValues(
       fieldDef,
-      fieldNodes[0],
-      exeContext.variableValues,
+      info.fieldNodes[0],
+      info.variableValues,
```

Comment:
Info should be opaque to the executor internals - it's only used to pass into resolvers. These two should be either passed in directly as arguments or pulled from exeContext